code/game/antagonist/station/contractor.dm

```diff
@@ -1,9 +1,9 @@
 // Inherits most of its vars from the base datum.
-/datum/antagonist/traitor
-	id = ROLE_TRAITOR
+/datum/antagonist/contractor
+	id = ROLE_CONTRACTOR
 	protected_jobs = list(JOBS_SECURITY, JOBS_COMMAND)
-	bantype = ROLE_TRAITOR
-	antaghud_indicator = "hudtraitor"
+	bantype = ROLE_CONTRACTOR
+	antaghud_indicator = "hudcontractor"
```

Comment:
this is used in .dmi file, if I recall correctly for ghosts antaghud and you didn't change the .dmi